doc/ref_model/chapters/chapter01.md

```diff
@@ -68,5 +39,54 @@ The document starts from the abstract and as it progresses it increasingly gets
   > **Audience**: This chapter is written for architects, developers and others who need to deploy infrastructure or develop applications.
 
+<a name="1.2"></a>
+## 1.2 Scope
+This document focuses on the documenting the higher level concepts that are needed to identify **Reference Model**. **Figure 1-1** below highlights its scope in more details.
+
+<p align="center"><img src="../figures/ch01_scope.png" alt="scope" title="Scope" width="100%"/></p>
+<p align="center"><b>Figure 1-1:</b> Scope of Reference Model</p>
+
+This document specifies:
+- **Cloud Infrastructure abstraction**: in context with how it interacts with the other components required to build a complete system that supports workloads deployed in Virtual Machines (VNF) or containers (CNF).
+  - **Cloud Infrastructure metrics & capabilities**: A set of metrics and capabilities for the cloud infrastructure which workloads require to perform telco scale network functions.
+  - **Infrastructure profiles catalogue**: A catalogue of standard profiles needed in order to abstract the infrastructure from workloads. With a limited and well-defined set of profiles with well understood characteristics, workload compatibility and performance predictability can be achieved.
+
+- Cloud Infrastructure Software and Hardware profiling
+  - **Cloud Infrastructure software profiles and configurations**: These are software profiles and configurations that map directly to the infrastructure profiles within the infrastructure profiles catalogue.
+  - **Cloud Infrastructure hardware profiles and configurations**: These are hardware profiles and configurations which are suitable for the defined cloud infrastructure software profiles & configurations.
+
+- Conformance and verification
+  - **Conformance programs**: This defines the requirements for certification and validation programs for both workloads and cloud infrastructure.
+  - **Test framework**: Provide test suites to allow conformance, certification, and verification of workloads and cloud infrastructure against the defined set of profiles.
+
+<a name="1.3"></a>
+## 1.3 Principles
+The Reference Model specification conform with the overall principles defined in the preface chapter [Cloud iNfrastructure Telco Taskforce](../../tech#2.0).
+
+<a name="1.4"></a>
+## 1.4 Definitions/Terminology
+To help guide the reader, a [CNTT Terminology](../../tech/glossary.md) section in [Cloud iNfrastructure Telco Taskforce] provides an introduction to the main terms used within this document and throughout the project in general. These definitions are, with a few exceptions, based on the [ETSI GR NFV 003 V1.5.1 (2020-01)](https://www.etsi.org/deliver/etsi_gr/NFV/001_099/003/01.05.01_60/gr_NFV003v010501p.pdf) definitions.  In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.
```

Comment:
"section in section in [Cloud iNfrastructure Telco Taskforce] provides .." -- maybe the intent was to include a reference but even then should it be CNTT documents?
last sentence two "avoid"
Aren't some of the definitions from cloud native foundation?

Reply:
@pgoyal01, we have to think how we will incorporate "Cloud iNfrastructure Telco Taskforce" tech document into RM as Chapter 0, as agreed between Rabi and Tom.  Then we will clean this up.  Topic for the RM Meeting.